index.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <title>Alison Zerbe</title>
+        <link href="styles/style.css" rel="stylesheet">
+        <!-- icon by AlfredoCreates.com -->
+        <!-- where do you put this attribution -->
+    </head>
+    <header>
+        <nav>
+            <!-- <img src="assets/triangle.png"> -->
+            <!-- <img src="assets/pyramid.svg" /> -->
+            <p>Alison Zerbe</p>
+            <ul>
+                <li><a href=about.html>About Me</a></li>
+                <li><a href="portfolio.html">Projects</a></li>
+                <li><a href="code-journal.html">Life At Ada</a></li>
+            </ul>
+        </nav>
+    </header>
+    <body>
+        <section id="banner">
+            <h1>Hello! I'm Alison!</p>
+            <h3>I am a... </h3>
+            <h3>in-progress software engineer, education addict,
+                language activist, globetrotter</h3>
+            <img src="assets/computer.svg" />
+            <img src="assets/education.svg" />
+            <img src="assets/thought_bubbles.svg" />
+            <img src="assets/globetrotter.svg" />
+        </section>
+
+        <section id="image">
+            <img src="https://unsplash.it/400">
+        </section>
+
+
+        <section id="more_info">
+            <article>
+                <a href="about.html">
+                    <img src="assets/hieroglyph.svg" />
+                    <span>About Me</span>
+                </a>
+            </article>
+            <article>
+                <a href="portfolio.html">
+                    <img src="assets/aztec_geometry.svg" />
+                    <span>Coding Projects</span>
+                </a>
+            </article>
+            <article>
+                <a href="code-journal.html">
+                    <img src="assets/pyramid_triangle.svg" />
+                    <span>My thoughts on Coding</span>    
+                </a>
+            </article>
+        </section>
+    </body>
+    <footer>
+        <section id="disclaimer">
+            <p>Copyright</p>
+        </section>
+        <section id="contact">
+            <!-- email link -->
+            <a href="#">Contact me</a>
+            <a href="#">Facebook</a>
+            <a href="#">LinkedIn</a>
+        </section>
+    </footer>
+</html>
```

Comment:
What Chris said - an HTML file has a `<head>` and a `<body>` and everything that's not in the header goes in the body. I Googled "semantic HTML outline cheatsheet" under Images, and found this site: http://www.hongkiat.com/blog/html-5-semantics/ - scroll down for a visual example of a website with HTML tags in the right places. 
Overall, I feel there are too many `section` tags, and I think you feel this way too, given that you gave each a unique ID to differentiate. I'm honestly not sure what is best practice here; this is something to reconsider and ask about.